docs/resources/users_permission.md

```diff
@@ -0,0 +1,85 @@
+# keycloak_users_permissions
```

Comment:
We'll need to add a page title to this doc page before this line.  Take a look at the other files within this folder for an example.